diff --git a/src/VBox/Main/xml/Settings.cpp b/src/VBox/Main/xml/Settings.cpp
index 046ae0d..0c99559 100644
--- a/src/VBox/Main/xml/Settings.cpp
+++ b/src/VBox/Main/xml/Settings.cpp
@@ -8933,7 +8933,8 @@ AudioDriverType_T MachineConfigFile::getHostDefaultAudioDriver()
     {
 # ifdef VBOX_WITH_AUDIO_PULSE
         /* Check for the pulse library & that the pulse audio daemon is running. */
-        if (RTProcIsRunningByName("pulseaudio") &&
+        if (RTProcIsRunningByName("pulseaudio") ||
+           RTProcIsRunningByName("pipewire-pulse") &&
             RTLdrIsLoadable("libpulse.so.0"))
             s_enmLinuxDriver = AudioDriverType_Pulse;
         else
