diff --git a/src/obs-ndi.cpp b/src/obs-ndi.cpp
index 283d64f..ac3ba4a 100644
--- a/src/obs-ndi.cpp
+++ b/src/obs-ndi.cpp
@@ -229,10 +229,7 @@ const NDIlib_v4 *load_ndilib()
 	if (!path.isEmpty()) {
 		locations << path;
 	}
-#if defined(__linux__) || defined(__APPLE__)
-	locations << "/usr/lib";
-	locations << "/usr/local/lib";
-#endif
+	locations << "@NDI@/lib";
 	for (QString location : locations) {
 		path = QDir::cleanPath(
 			QDir(location).absoluteFilePath(NDILIB_LIBRARY_NAME));
